Whitelist-dApp.md

```diff
@@ -158,7 +158,7 @@ Now open the hardhat.config.js file, we would add the `goerli` network here so t
 
 ```js
 require("@nomicfoundation/hardhat-toolbox");
-require("dotenv").config({ path: ".env" });
+require("dotenv").config({ path: "hardhat-tutorial/.env" });
```

Comment:
Hi, 
The `.env` file should be inside the `hardhat-tutorial` folder as mentioned in the README.
The file you are editing is `hardhat.config.js` which is in the root of the folder `hardhat-tutorial` (the same level as `.env` and therefore doesn't need any value inside path. 

![FolderStructure](https://i.imgur.com/E074gzA.png)

Technically even `require("dotenv").config()` should work.